src/main/target/STM32F411DISCOVERY/target.h

```diff
@@ -0,0 +1,120 @@
+/*
+ * This file is part of Cleanflight and Betaflight.
+ *
+ * Cleanflight and Betaflight are free software. You can redistribute
+ * this software and/or modify this software under the terms of the
+ * GNU General Public License as published by the Free Software
+ * Foundation, either version 3 of the License, or (at your option)
+ * any later version.
+ *
+ * Cleanflight and Betaflight are distributed in the hope that they
+ * will be useful, but WITHOUT ANY WARRANTY; without even the implied
+ * warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define TARGET_BOARD_IDENTIFIER "S411" // STM Discovery F411
+#define USBD_PRODUCT_STRING     "DISCF411"
+
+#define USE_SENSOR_NAMES
+
+#define CURRENT_TARGET_CPU_VOLTAGE 3.0
```

Comment:
Have you confirmed that this is actually correct, or is this something you copied from the STM32F3DISCOVERY target?

Reply:
Oh, I forgot to check that, it is indeed copied. I think it's supposed to be 5V, but is that define necessary in that case at all?

Reply:
It's definitely not 5V. But the default, and what almost all boards are using, is 3.3V. You'll probably have to check that with the STM documentation for the Discovery board.

Reply:
Board is 5V, CPU is 2.7-3.6V, so 3.3V target if I understand that correctly. Is that define required for that?

Reply:
No.